server/service/src/stocktake/update.rs

```diff
@@ -361,6 +363,82 @@ fn generate_stock_in_out_or_update(
     })
 }
 
+fn log_stock_changes(
```

Comment:
This file is getting really big.... Would really like a refactor into generate.rs/validate.rs files....

Reply:
10000000% agree yeehaw